app/src/main/res/layout/activity_stand_alone.xml

```diff
@@ -60,15 +60,22 @@
           android:layout_marginStart="5dp"
           android:layout_marginLeft="5dp"
           android:background="@android:color/holo_blue_dark"
-          android:text="@string/btn_advanced_native"/>
+          android:text="@string/btn_advanced_native" />
       </LinearLayout>
     </HorizontalScrollView>
-    <LinearLayout
-      android:id="@+id/AdLayout"
+    <com.criteo.publisher.CriteoBannerView
+      android:id="@+id/criteoBannerView"
       android:layout_width="match_parent"
```

Comment:
What do you think about using a fixed size ? So it is possible to have both banner and native without changing the visibility.